src/components/paragraph/stanford-spacer.tsx

```diff
@@ -1,19 +1,15 @@
 import {Maybe} from "@/lib/gql/__generated__/drupal.d"
+import {clsx} from "clsx"
 
 const StanfordSpacer = ({size}: {size?: Maybe<"spacer-reduced" | "spacer-minimal" | string>}) => {
-  let height
-  switch (size) {
-    case "spacer-reduced":
-      height = "min-h-[20px]"
-
-      break
-    case "spacer-minimal":
-      height = "min-h-[10px]"
-
-      break
-    default:
-      height = "min-h-[40px]"
```

Comment:
there's a bug where the spacer results in `min-h-[40px]` regardless of what the content author selects.